article/related-work.tex

```diff
@@ -10,13 +10,51 @@ \subsection{Light GCN}
 \end{equation}
 where $W_1$ and $W_2 \in R^{d' \times d}$ are weight matrices and $d'$ is the transformation size.
 $\frac{1}{\sqrt{|\mathcal{N}_u||\mathcal{N}_i|}}$ is the graph Laplacian norm, where $\mathcal{N}_u$ and $\mathcal{N}_i$ are the set of neighbors of user $u$ and item $i$.
-This is done with the purpose of calculating how much the item contributes to the users preference \cite{NGCF_2019}. % WTF is this shit????
+This is done with the purpose of calculating how much the item contributes to the users preference \cite{NGCF_2019}.
 LightGCN criticize the use of the weight matrices $W_1$ and $W_2$ as not being useful for CF as each user-item interaction graph only has the ID as input and it has no semantic value \cite{lightgcn}.
 Also within the second component the message aggregation is implemented as:
 \begin{equation}
     e_{u}^{(l)} = \mbox{LeakyReLU}(m^{(l)}_{u \leftarrow u} + \sum^{}_{i \in \mathcal{N}} m^{(l)}_{u \leftarrow i}),
 \end{equation}
 where $e_{u}^{(l)}$ are the embeddings of user $u$ at layer $l$.
-LeakyReLU is the activation function chosen in NGCF to allow encoding positive and small negative signals. % WHAT DOES THIS EVEN MEAN???
+LeakyReLU is the activation function chosen in NGCF to allow encoding positive and small negative signals.
 LightGCN shows that NGCF will perform better if the feature transformation is removed, and that the activation function has small effect when the feature transformation is included, but if feature transformation is disabled the activation function will have a negative impact on performance.
 Light GCN also shows that NGCF will significantly improve if both the activation function and feature transformation are removed \cite{lightgcn}.
+
+\subsection{Price-aware Recommendation with Graph Convolutional Networks}
+Yu Zheng et al. develops an effective method called \textit{Price-aware User Preference-modeling (PUP)} that is used to create recommendations with focus on the price factor \cite{Priceaware}.
+There are two main challenges.
+The first is that the preferences of a user on item price are unknown, and can only be seen implicitly by previous purchases.
+The second challenge is that the price of an item largely depends on what category the item is within.
+The first problem they propose a model that creates a relationship between user-to-item and item-to-price using a Graph Convolutional Network, so that they can propagate the influence of price onto the users.
```

Comment:
```suggestion
For the first problem they propose a model that creates a relationship between user-to-item and item-to-price using a Graph Convolutional Network, so that they can propagate the influence of price onto the users.
```